lib/sorbet-rails/model_plugins/active_record_assoc.rb

```diff
@@ -73,9 +73,16 @@ def populate_single_assoc_getter_setter(assoc_module_rbi, assoc_name, reflection
         level but **nullable** at the DB level.\n Add a constraint at the DB level
         (using `null: false` and foreign key constraint) to ensure it is enforced.".squish!
     elsif !rails_required_config && db_required_config
-      puts "Note: belongs_to association #{reflection.name} is specified as not-null at the
-        DB level but **optional** at the application level.\n Add a constraint at the app level
-        (using `optional: false`) as a validation hint to Rails.".squish!
+      if habtm?
+        puts "Note: belongs_to association #{reflection.name} is specified as not-null at the
```

Comment:
These messages try to be helpful in terms of suggesting good configuration. I think we likely don't want to remove `not-null` constraint at the DB level. Perhaps we can skip logging for this case.

Reply:
this warning would have been useful for my employer's codebase but i don't feel that strongly about it. 

fwiw i'm also working on updating the rails docs about `habtm`

Reply:
or alternately i can remove the suggestion about removing the not null constraint and leave the other suggestion (which is what we did locally)?

Reply:
Yeah, let's remove the suggestion about "removing not null constraint"